lib/streamlit/elements/media.py

```diff
@@ -118,6 +121,9 @@ def audio(
               e.g. ``timedelta(seconds=70)``.
         loop: bool
             Whether the audio should loop playback.
+        autoplay: bool
+            Whether the audio should start playing automatically.
+            Browsers will not autoplay audio files if the user has not interacted with the page yet.
```

Comment:
nit: should we add the `, for example by clicking on the page while it loads.` here for consistency as well?

Reply:
I didn't spot that in my review. Fixed on 710a10d